test/machineslib.py

```diff
@@ -40,6 +40,8 @@ def performAction(self, vmName, action, checkExpectedState=True, connectionName=
             b.wait_in_text("#vm-{0}-{1}-state".format(vmName, connectionName), "Running")
         if action == "forceOff":
             b.wait_in_text("#vm-{0}-{1}-state".format(vmName, connectionName), "Shut off")
+        if action == "off":
+            b.wait_in_text("#vm-{0}-{1}-state".format(vmName, connectionName), "Shut off")
 
```

Comment:
You defined just new name for the same action, I don't think we need to do that. But more curiously, why force is represented just as 'Shut off' in those tests?